src/main/java/com/pnudev/springonlinedelivery/Order_U.java

```diff
@@ -0,0 +1,15 @@
+package com.pnudev.springonlinedelivery;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+
+@Table
+@Entity
+public class Order_U {
```

Comment:
What exactly is the purpose of this entity? If it's an intermediate table for `OneToMany`/`ManyToMany` relation then it should be done via the Hibernate annotations (e.g. https://www.baeldung.com/hibernate-one-to-many)

Reply:
**MenuItemOrder** is an intermediate table, I named this entity this way because if you leave the "Order", an DDL error will occur

Reply:
By the way, I tried to implement these relations, but not very successfully